exampleCourse/questions/demo/workspace/rstudio/question.html

```diff
@@ -10,13 +10,11 @@
 - a numeric `nn` with value 3.1415
 - a text variable `cc` with text `hello, world!` including the comma an exclamation
 
-You can press the button below to launch a demo RStudio workspace.
+Press the button below to launch an RStudio workspace. The object above must be created in the `student.R` file inside the workspace.
```

Comment:
It seems RStudio does not auto-save, so a reminder to save the file in RStudio may be a good idea.

Reply:
Sure, will add that here.

Reply:
Removed here https://github.com/PrairieLearn/PrairieLearn/pull/7368/commits/e24fa2667373b1d5f305b159a179d02ed7e799b8